# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1468593468 14400
#      Fri Jul 15 10:37:48 2016 -0400
# Node ID b017351e92e1e744defe194eef27bf4a8a77861f
# Parent  c3b5f26721e11ccf159e0e47017ff7c51fd9ae42
* kpse.cc (kpse_truncate_filename): Use namespace qualifier for static dir_entry function.

diff --git a/liboctave/util/kpse.cc b/liboctave/util/kpse.cc
--- a/liboctave/util/kpse.cc
+++ b/liboctave/util/kpse.cc
@@ -140,17 +140,17 @@ kpse_truncate_filename (const std::strin
 
   for (size_t i = 0; i < len; i++)
     {
       if (IS_DIR_SEP (name[i]) || IS_DEVICE_SEP (name[i]))
         {
           /* At a directory delimiter, reset component length.  */
           c_len = 0;
         }
-      else if (c_len > dir_entry::max_name_length ())
+      else if (c_len > octave::sys::dir_entry::max_name_length ())
         {
           /* If past the max for a component, ignore this character.  */
           continue;
         }
 
       /* Copy this character.  */
       ret[ret_len++] = name[i];
       c_len++;
